install_config/default_imagestreams_templates.adoc

```diff
@@ -0,0 +1,420 @@
+= Default Image Streams and Templates
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+// tag::firststeps[]
+
+== Overview
+
+ifdef::openshift-enterprise[]
+Your OpenShift installation includes a useful set of Red Hat-provided
+link:../architecture/core_concepts/builds_and_image_streams.html#image-streams[image
+streams] and link:../architecture/core_concepts/templates.html[templates] to
+make it easy for developers to create new applications. By default, the
+link:../install_config/install/quick_install.html[quick installation] and
+link:../install_config/install/advanced_install.html[advanced installation]
+methods automatically create these sets in the *openshift* project, which is a
+default global project to which all users have view access.
+endif::[]
+ifdef::openshift-origin[]
+You can populate your OpenShift installation with a useful set of Red
+Hat-provided
+link:../../architecture/core_concepts/builds_and_image_streams.html#image-streams[image
+streams] and link:../../architecture/core_concepts/templates.html[templates] to
+make it easy for developers to create new applications. By default, the
+link:advanced_install.html[advanced installation] method automatically creates
+these sets in the *openshift* project, which is a default project to which all
+users have view access.
+endif::[]
```

Comment:
per comment above...why aren't we just linking here for the origin docs, instead of linking to first_steps?